packages/next/src/index.ts

```diff
@@ -408,8 +407,6 @@ export const build: BuildV2 = async ({
   }
 
   const env: typeof process.env = { ...spawnOpts.env };
-  const memoryToConsume = Math.floor(os.totalmem() / 1024 ** 2) - 128;
-  env.NODE_OPTIONS = `--max_old_space_size=${memoryToConsume}`;
```

Comment:
Does it make sense to set this if env.NODE_OPTIONS is not already defined?

Reply:
No, I don't think so. It should be assigned in a build container or project settings.